exporters/otlp-http/trace/src/main/java/io/opentelemetry/exporter/otlp/http/trace/OtlpHttpSpanExporter.java

```diff
@@ -85,16 +84,14 @@
   @Override
   public CompletableResultCode export(Collection<SpanData> spans) {
     spansSeen.add(spans.size());
-    ExportTraceServiceRequest exportTraceServiceRequest =
-        ExportTraceServiceRequest.newBuilder()
-            .addAllResourceSpans(SpanAdapter.toProtoResourceSpans(spans))
-            .build();
+
+    TraceRequestMarshaler exportRequest = TraceRequestMarshaler.create(spans);
```

Comment:
NIT: rename variable to `spansMarshaller` ?

Reply:
Since it corresponds to the export request, I like that name a bit better